oak_functions/client/src/main.rs

```diff
@@ -39,6 +38,8 @@ pub struct Opt {
         default_value = "http://localhost:8080"
     )]
     uri: String,
+    #[structopt(long, help = "request payload")]
+    request: String,
```

Comment:
Seeing that this is required, I assume the flag must be passed in the example as well.